content/blog/announcing-direct-tf-modules/index.md

```diff
@@ -197,12 +198,35 @@ Duration: 2m26s
 The infrastructure has now provisioned and the corresponding Terraform state is stored securely inside the Pulumi
 state, which can be verified with `pulumi stack export`.
 
-The above program is very simple. To take it further, check out
+The above program is a very basic use case. To take it further, check out
 [examples](https://github.com/pulumi/pulumi-terraform-module/tree/main/examples) for more realistic programs showcasing
 features such as computing subnets dynamically with Pulumi `aws.getAvailabilityZonesOutput` function or passing the
 results of the VPC module to an EKS module.
 
-// TODO show-case cross-configuring the provider with a given AWS region
+### Passing through existing configuration
+
+As an example how the new Pulumi Terraform Module can interact with existing Pulumi programs, we've made it possible
```

Comment:
This one's interesting there's a lot going on here and it can be confusing that both Pulumi and TF providers are involved and that you need to configure both. Perhaps find a quick way to call this out? 

With the caveat of it maybe being too verbose, for example start by showing how you can configure the AWS region for the TF provider using the explicit provider. And then continue with saying if you already have a *Pulumi* AWS provider you can reuse the region and everything else from that provider.